linux-cachyos/PKGBUILD

```
@@ -237,6 +252,15 @@ if [ -n "$_build_nvidia_open" ]; then
              "${_patchsource}/misc/nvidia/0006-nvidia-drm-Set-FOP_UNSIGNED_OFFSET-for-nv_drm_fops.f.patch")
 fi
 
+# Use generated AutoFDO Profile
+if [ -n "$_autofdo" ] && [ -n "$_autofdo_profile_name" ]; then
+    if [ -e "$_autofdo_profile_name" ]; then
+        source+=("$_autofdo_profile_name")
+    else
+        _die "You have specified _autofdo_profile_name, but file it refers to doesn't exist."
```

```suggestion
        _die "Failed to find file $_autofdo_profile_name."
```
Wonder if this would be better?